src/Core/Domain/Discount/ValueObject/DiscountType.php

```diff
@@ -28,11 +28,11 @@
 
 class DiscountType
 {
-    public const CART_DISCOUNT = 'cart_discount';
-    public const PRODUCTS_DISCOUNT = 'products_discount';
+    public const CART_DISCOUNT = 'cart_level';
+    public const PRODUCTS_DISCOUNT = 'product_level';
     public const FREE_GIFT = 'free_gift';
     public const FREE_SHIPPING = 'free_shipping';
-    public const ORDER_DISCOUNT = 'order_discount';
+    public const ORDER_DISCOUNT = 'order_level';
```

Comment:
```suggestion
    public const ORDER_LEVEL = 'order_level';
```
It would make sense to keep the const names and their values synced then